src/components/UI/Icon/Icon.tsx

```diff
@@ -148,25 +148,25 @@ export const IconResolverAddress: React.FunctionComponent<SVGProps<SVGElement>>
   );
 };
 
-export const IconSuccess: React.FunctionComponent<SVGProps<SVGElement>> = () => {
+export const IconSuccess: React.FunctionComponent<SVGProps<SVGElement>> = ({ className }) => {
   return (
     <svg
-      className="mx-auto mt-10"
-      width="56"
-      height="56"
-      viewBox="0 0 56 56"
+      className={`${className}`}
```

Comment:
```suggestion
      className={`${className} ? ${className} : ""`}
```

in the case of no className, should not be `undefined` set. this suggestion doesn't look nice also, if there's a better way to cater to `undefined` cases, go ahead

Reply:
ok, i created a util function that checks for undefinedClassName, would this be better?